conftest.py

```diff
@@ -27,6 +27,21 @@
 FULL_BENCHMARK_MARK = "full_benchmark"
 
 
+def print_session_id():
+    # s3 is the only storage provider that uses the SESSION_ID prefix
+    # if it is enabled, print it out after all tests finish
+    print("\n\n")
+    print("----------------------------------------------------------")
+    print("Testing session ID: %s" % SESSION_ID)
+    print("----------------------------------------------------------")
+    print("\n\n")
+
+
+
+# before tests start print session ID
+print_session_id()
```

Comment:
Why before _and_ after?

Reply:
because before you want to be able to follow the tests while its happening in the s3 bucket, and then after because if you use the `-s` flag it will print a LOT of stuff. so in case you forgot to copy the ID then it will print it again at the end